src/room/PTTButton.tsx

```diff
@@ -130,12 +133,32 @@ export const PTTButton: React.FC<Props> = ({
       );
     };
   }, [onWindowMouseUp, onWindowTouchEnd, onButtonTouchStart, buttonRef]);
+
+  const { shadow } = useSpring({
+    shadow: (Math.max(activeSpeakerVolume, -70) + 70) * 0.5,
+    config: {
+      clamp: true,
+      tension: 300,
+    },
+  });
+  const shadowColor = showTalkOverError
+    ? "rgba(255, 91, 85, 0.2)"
+    : "rgba(13, 189, 139, 0.2)";
```

Comment:
Could we use a CSS variable? I think they should be supported natively on all our target browsers, so I think it ought to be fine? Otherwise, maybe just have them as consts with a comment saying what they are?